scout/adapter/mongo/query.py

```diff
@@ -114,15 +114,19 @@ def build_query(self, case_id, query=None, variant_ids=None, category='snv'):
             cadd_query = {'cadd_score': {'$gt': float(cadd)}}
             logger.debug("Adding cadd_score: %s to query" % cadd)
 
-            if query.get('cadd_inclusive') == 'yes':
-                cadd_query = {
-                    '$or': [
-                        cadd_query,
-                        {'cadd_score': {'$exists': False}}
-                    ]}
-                logger.debug("Adding cadd inclusive to query")
+            if query.get('cadd_inclusive') is not None:
```

Comment:
I don't really follow. First you check that `cadd_inclusive != None`. 
Then you check if `cadd_inclusive == True`... Why not check that from the beginning? And also cadd_inclusive is set to 'yes' in `query` so this would not work if I understand everything right

Reply:
After parsing the query, it will be True rather than 'y' as it appears on the html query. I haven't tested the undefined part, but at least the "cadd_inclusive" part is not on the http request line if the box is not checked..

Reply:
It seems the test is not correct?

Reply:
Couldn't you skip some lines by only checking `if cadd_inclusive is True:` at the top?